go/chat/utils/utils.go

```diff
@@ -1122,7 +1122,26 @@ func presentPaymentInfo(ctx context.Context, g *globals.Context, msgID chat1.Mes
 	switch typ {
 	case chat1.MessageType_SENDPAYMENT:
 		body := msg.MessageBody.Sendpayment()
-		return g.PaymentLoader.Load(ctx, convID, msgID, msg.SenderUsername, body.PaymentID)
+		if g.StellarLoader != nil {
```

Comment:
Any chance I can convince you to make a `DummyStellarLoader` instead of checking for `nil`? I just get nervouse with `nil` and interfaces.